app/common/elasticsearchclient.py

```diff
@@ -1021,13 +994,6 @@ def get_associations(self,
                 }
             }
 
-        # print "------------"
```

Comment:
I need this @peatroot to help me with the debug process from time to time

Reply:
Added back in